### DISCUSSION TITLE
SBLGNT Relicensed as CC-BY

### DISCUSSION THREAD

**Discussion by prust** [openenglishbible/Open-English-Bible#355] [General]
@russellallen: I just stumbled on this mention that the SBL's Greek New Testament was CC-BY licensed: https://github.com/RickBrannan/bible-comparison#license

I followed the links and found [this commit](https://github.com/LogosBible/SBLGNT/commit/1db39495) from Dec 2022 that appears to be when it was relicensed. If I recall correctly, the Open English Bible project was avoiding reliance on the SBL GNT due to licensing, but perhaps this is no longer an issue?

**Comment by openenglishbible** [comment]
Thanks for pointing this out, I hadn't realised that had happened.

I'll see if I can find an easy way to compare it with W&H...

I'd really like the NA28 under a liberal license, but so far Santa hasn't obliged despite my nice letter to him.

**Comment by RobH123** [comment]
See if https://GreekCNTR.org/collation/ is useful for comparing. (I understand that Alan Bunning's data was used by the SBL GNT project, and would recommend his open-licensed Statistical Restoration GNT as better anyway. See http://jbtc.org/v28/TC-2023_Bunning.pdf.)

P.S. Note that capitalisation, punctuation, and accents aren't included in the NA text above for copyright reasons (so that the NA can't be recreated from those pages).

**Comment by openenglishbible** [comment]
Thanks, that's very interesting. 

(I will say in defence of WH that it is a very good critical text, and very close to both NA and (from my initial cursory reading) Bunning's SR.)

**Comment by RobH123** [comment]
Yes, I agree. What's different is that a group can easily be inconsistent (or unintentionally biased) in making decisions in one place from in another. The SR is done by an algorithm considering the estimated/assigned dates for the manuscripts, etc. And because it's both free and modern, then it can be a good option. Alan Bunning spent two decades transcribing manuscripts and writing analysis software. (Much of that hadn't been done by anyone before then.) His website is the most useful that I've found for comparing texts. And his apparatus is complete (and better, many say) as well. His work is ongoing to produce a lexicon, and also to add later manuscripts dated after 400AD (esp. church fathers) which are not yet included in the calculations.

P.S See column 29 in 2 Peter 3:10 (https://greekcntr.org/collation/?v=61003010) to see how new Greek words (NA) can be invented these days.

**Comment by openenglishbible** [comment]
That site is really quite nice for showing differences at a verse level.
